core/input/input_map.cpp

```diff
@@ -353,6 +353,7 @@ static const _BuiltinActionDisplayName _builtin_action_display_names[] = {
     { "ui_text_scroll_down",                           TTRC("Scroll Down") },
     { "ui_text_scroll_down.OSX",                       TTRC("Scroll Down") },
     { "ui_text_select_all",                            TTRC("Select All") },
+    { "ui_text_select_word_under_cursor",              TTRC("Select Word Under Cursor") },
```

Comment:
Please use `caret` instead of `cursor`, this goes for the rest of the code as well.

Reply:
Methods in TextEdit use `cursor` instead of `caret` (e.g. `get_word_under_cursor()`). Should I also change the method name in TextEdit?